src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -58,6 +58,16 @@
 from sentry.utils import json
 
 STATUSES = {"resolved": "resolved", "ignored": "ignored", "unresolved": "re-opened"}
+SUPPORTED_PROVIDER_TO_COMMIT_URL = {
+    "github": "{base_url}/commit/{commit_id}",
+    "integrations:github": "{base_url}/commit/{commit_id}",
+    "integrations:github_enterprise": "{base_url}/commit/{commit_id}",
+    "integrations:vsts": "{base_url}/commit/{commit_id}",
+    "integrations:gitlab": "{base_url}/commit/{commit_id}",
+    "bitbucket": "{base_url}/commits/{commit_id}",
+    "integrations:bitbucket": "{base_url}/commits/{commit_id}",
```

Comment:
these are based off the repo providers added to `bindings` within `src/sentry/integrations/` and `src/sentry_plugins/`